docs/pages/repo/docs/installing.mdx

```diff
@@ -0,0 +1,85 @@
+---
+title: Installing Turborepo
+description: Learn how to install Turborepo for use with your repository
+---
+
+import Callout from "../../../components/Callout";
+import { Tabs, Tab } from '../../../components/Tabs'
+
+# Install Turborepo
+
+`turbo` works with [Yarn](https://classic.yarnpkg.com/lang/en/), [npm](https://www.npmjs.com/), and [pnpm](https://pnpm.io/) on the following operating systems:
+
+- macOS darwin 64-bit (Intel), ARM 64-bit (Apple Silicon)
+- Linux 64-bit, ARM 64-bit
+- Windows 64-bit, ARM 64-bit
+
+<Callout>
+  Note: Linux builds of `turbo` link against `glibc`. For Alpine Docker environments, you will need to ensure libc6-compat is installed as well, via `RUN apk add --no-cache libc6-compat`
+</Callout>
+
+## Install Globally
+
+A global install of `turbo` can be used in any project, and enables automatic workspace
+selection based on the directory where you run `turbo`.
+
+<Tabs items={['npm', 'yarn', 'pnpm']} storageKey="selected-pkg-manager">
+  <Tab>
+    ```bash
+    npm install turbo --global
+    ```
+  </Tab>
+  <Tab>
+    ```bash
+    yarn global add turbo
+    ```
+  </Tab>
+  <Tab>
+    ```bash
+    pnpm install turbo --global
+    ```
+  </Tab>
+</Tabs>
+
+Once you have a globally installed copy of `turbo`, you will be able to run directly from workspace
+directories.
+
+```bash
+cd <repository root>/apps/docs
+turbo build
+```
+
+is equivalent to
+
+```bash
+cd <repository root>
+turbo build --filter=docs
+```
+
+## Install Per Repository
+
+You may wish to pin the version of Turborepo used within a repository, especially if you are
+collaborating with other developers. In that case, add `turbo` as a dev dependency at the root
+of the repository:
+
+<Tabs items={['npm', 'yarn', 'pnpm']} storageKey="selected-pkg-manager">
+  <Tab>
+    ```bash
+    npm install turbo --dev
+    ```
+  </Tab>
+  <Tab>
+    ```bash
+    yarn add turbo --dev --ignore-workspace-root-check
```

Comment:
I don't think we should recommend using `--ignore-workspace-root-check` if we're also instructing to install "...as a dev dependency at the root of the repository:"

Reply:
I'm not sure I understand. `yarn` will report an error if you don't add that flag. You use that flag to dependencies "at the root of the repository": https://classic.yarnpkg.com/en/docs/cli/add#toc-yarn-add-ignore-workspace-root-check-w

Reply:
Ah I've never actually seen these full commands, I've only seen their shorthand (`-w` for pnpm and `-W` for yarn). 

All good.